# This srv file defines the service
# for generating the IK/configs for object start positions in advance

---
bool success